nuxeo-ai-config/pom.xml

```diff
@@ -25,10 +29,6 @@
       <groupId>org.nuxeo.ecm.automation</groupId>
       <artifactId>nuxeo-automation-core</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.nuxeo.ai</groupId>
-      <artifactId>nuxeo-ai-core</artifactId>
-    </dependency>
```

Comment:
It's needed for a dependency on EnrichmentTestFeature

Reply:
i removed it cause i duplicated it previously in the pom

Reply:
But you're right actually cause i didnt set the test-jar hehe. Thanks!